tests/Tests/500__jupyterhub/test-jupyterlab-notebook.robot

```diff
@@ -54,3 +54,26 @@ Can Launch Python3 Smoke Test Notebook
   #Get the text of the last output cell
   ${output} =  Get Text  (//div[contains(@class,"jp-OutputArea-output")])[last()]
   Should Not Match  ${output}  ERROR*
+
+Verify A Default Image Is Provided And Starts Successfully
+    [Documentation]    Verify that, if a user doesn't explicitly select any jupyter image
+    ...    a default one is selected and it can be spawned successfully
+    [Tags]    Sanity
+    ...       ODS-469
+    @{user_data} =    Get Previously Selected Notebook Image Details
+    Launch JupyterHub Spawner From Dashboard
```

Comment:
I had to revert this PR because the way the `Verify A Default Image Is Provided And Starts Successfully` TC is currently set up is wrong. 
When the execution of `Can Launch Python3 Smoke Test Notebook` ends the browser will be open and in the JupyterLab Interface. This TC expects to be able to launch JupyterHub from the RHODS dashboard, and it will fail because it is currently in a different page (JL). 

This can be fixed in two ways that I can think of:
Update the end of `Can Launch Python3 Smoke Test Notebook` or the beginning of `Verify A Default Image Is Provided And Starts Successfully` to 1) Clean Up the server, 2) Close the Server, 3) Go Back to the RHODS dashboard

Another valid option would be to move `Verify A Default Image Is Provided And Starts Successfully` in a different `.robot` file, with its own setup and teardown keywords. That way no further changes would be needed to `test-jupyterlab-notebook.robot`

Reply:
@prudhvi-165 @bdattoma @jgarciao ^^